source/ncengine/graphics2/diligent/ShaderFactory.cpp

```diff
@@ -24,6 +24,31 @@ auto ReadShaderFile(std::string_view filePath) -> std::vector<char>
     return buffer;
 }
 
+auto ShaderFactory::MakeShaderFromPath(std::span<const char> path,
```

Comment:
What's this guy for? We should have tests covering usage of `ReadShaderFile()` + `MakeShaderFromSource()` to make a shader from source file.

Reply:
Missed that, will fix now